map_model/src/edits/mod.rs

```diff
@@ -38,6 +39,7 @@ pub struct MapEdits {
     /// Derived from commands, kept up to date by update_derived
     pub changed_roads: BTreeSet<RoadID>,
     pub original_intersections: BTreeMap<IntersectionID, EditIntersection>,
+    pub original_crosswalks: BTreeMap<IntersectionID, EditCrosswalks>,
```

Comment:
I made all the changes at the map model layer pretty mechanically. There might be some long-term problem that bites us later; I haven't thought about things super carefully here